crates/rooch-sequencer/src/actor/transaction_list.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use rooch_types::transaction::TypedTransaction;
+use rooch_types::H256;
+
+// TODO: test
+pub struct MemTransaction {
+    pub tx: TypedTransaction,
+    pub next: Option<Box<MemTransaction>>,
+}
+
+pub struct MemTransactionList {
+    pub head: Option<Box<MemTransaction>>,
+}
+
+impl MemTransactionList {
+    pub fn add_transaction(&mut self, transaction: TypedTransaction) {
+        let mut new_node = Box::new(MemTransaction {
```

Comment:
If use a linked list for mock storage, it means that even if use hash as a key to query, it needs to traverse the linked list

Reply:
We should define a transaction storage trait and implement memory storage. The memory storage is just a mock storage, we don't need to consider memory capacity and performance issues.